### ISSUE TITLE
Provide executable for many archs

### ISSUE THREAD

**Issue by vinceferro** [paritytech/substrate#11205]
Hey team,
Thanks for the awesome piece of technology!

I'm here to propose you to ship either a multi-arch docker image or to ship executable for many archs as it would make the adoption much easier given the rise of the Apple Silicon. I am currently trying to get the multi-arch docker image in place, but I am struggling a bit with some buildx issues.

Maybe you already have something along those lines? In that case please share a pointer to that ;)

**Comment by vinceferro** [comment]
Yes exactly, the binary for the node. We use it in the process of developing our smart contract, and unfortunately we are on different archs.

**Comment by vinceferro** [comment]
Yes exactly, the binary for the node. We use it in the process of developing our smart contract, and unfortunately we are on different archs.

**Comment by bkchr** [comment]
@athei can you help here? Is canvas better for developing contracts and do you may provide some more architectures?

**Comment by athei** [comment]
For developing smart contracts one should use https://github.com/paritytech/substrate-contracts-node . It uses instant consensus for ease of development. We also have Linux and Mac releases there. However, the Mac release in Intel only for now. We should adapt the CI to also include Apple Silicon.
